### ISSUE TITLE
aria-describedby referenced to empty element is not shown as "needs review"

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2149]
The intended fix for issue #1151 is not working as expected. Case-1 in the below examples does not show up as "needs review'.

Demo: https://codepen.io/iamrafan/pen/abOrzVb?editors=1000

```aria-describedby``` referenced to an empty ```<span>``` is now not a failure but it does not show up as 'needs review' as well. 

<pre><code>axe-core version: 3.5.2-canary.37be85f
axe-Coconut version: 4.4.1</code></pre>

**Comment by straker** [comment]
The code is working as expected from that issue. The original issue was that axe errored when the element was missing but has always passed if the element exists (even when empty). So the change allows non-existent elements to not fail.

Are you saying you'd like axe-core to flag empty aria-describedby elements as needs review?

**Comment by iamrafan** [comment]
@straker Thanks for the clarification. 

> Are you saying you'd like axe-core to flag empty aria-describedby elements as needs review?

I am good either way
